plugins/com.google.cloud.tools.eclipse.test.logback/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: com.google.cloud.tools.eclipse.test.logback
+Bundle-Version: 0.1.0.qualifier
+Fragment-Host: ch.qos.logback.classic
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ActivationPolicy: lazy
+Import-Package: org.slf4j.impl
+Eclipse-ExtensibleAPI: true
```

Comment:
`ExtensibleAPI` is a directive for PDE to allow `.test.logback` to see other fragments extending the same bundle host. This line is from an earlier experiment and isn't necessary; I'll remove it.